<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_film</class>
 <widget class="QDialog" name="gestion_film">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gestion_films</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(:/caca.jpg);

</string>
  </property>
  <widget class="QTabWidget" name="chercher">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>721</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
*{
background:#aaaa7f;
font-family:century gothic;
font-size:15px;

}
QGroupBox
{
background:#333;
background-image:url(:/caca.jpg);
}
#ajouter_film
{
color:white;
background:#a50000;
border-radius:60px;
}
#ajouter_film:hover
{
color:black;
border-radius:60px;
background:#000000;
}
#pushButton_supprimer
{
color:black;
background:#a50000;
border-radius:60px;
}
#pushButton_supprimer:hover
{
color:white;
border-radius:60px;
background:#000000;
}
#pushButton
{
color:black;
background:#a50000;
border-radius:60px;
}
#pushButton:hover
{
color:white;
border-radius:60px;
background:#000000;
}
#pushButton_modifier
{
color:black;
background:#a50000;
border-radius:60px;
}
#pushButton_modifier:hover
{
color:white;
border-radius:60px;
background:#000000;
}
#pushButton_3
{
color:black;
background:#a50000;
border-radius:60px;
}
#pushButton_3:hover
{
color:white;
border-radius:60px;
background:#000000;
}
#pushButton_chercher
{
color:black;
border-radius:60px;
background:#99007f;
}
#pushButton_chercher:hover
{
color:white;
background:#ff5500;
border-radius:60px;
}
#pushButton_chercher_2
{
color:black;
border-radius:60px;
background:#99007f;
}
#pushButton_chercher_2:hover
{
color:white;
background:#ff5500;
border-radius:60px;
}
#pushButton_modif
{
color:black;
border-radius:60px;
background:#99007f;
}
#pushButton_modif:hover
{
color:white;
background:#ff5500;
border-radius:60px;
}
#imprimer
{
color:black;
border-radius:60px;
background:#99007f;
}
#imprimer:hover
{
color:white;
background:#ff5500;
border-radius:60px;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>AJOUTER</string>
    </attribute>
    <widget class="QLabel" name="label_nom_film">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NOM_FILM:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom_film">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_duree_film">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>DUREE_FILM:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_language_film">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>LANGUAGE_FILM:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_type_film">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>170</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TYPE_FILM:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_duree_film">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_language_film">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_type_film">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter_film">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
*{
font-family:century gothic;
font-size:24px;

}
QGroupBox
{
background:#333;
background-image:url(:/caca.jpg);
}
#ajouter_film
{
color:black;
background:#a50000;
border-radius:60px;
}
#ajouter_film:hover
{
color:white;
border-radius:60px;
background:#000000;
}
#supprimer
{
color:white;
background:#000000;
border-radius:60px;
}
#supprimer:hover
{
color:black;
border-radius:60px;
background:#a50000;
}
#pushButton_3
{
color:white;
background:#000000;
border-radius:60px;
}
#pushButton_3:hover
{
color:black;
border-radius:60px;
background:#a50000;
}
</string>
     </property>
     <property name="text">
      <string>AJOUTER</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_id_film">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID_FILM:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id_film">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_chercher">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>110</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>CHERCHER</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>160</y>
       <width>371</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>SUPPRIMER</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ID_FILM:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_supp">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_supprimer">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>70</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
*{
font-family:century gothic;
font-size:24px;

}
QGroupBox
{
background:#333;
background-image:url(:/caca.jpg);
}
#ajouter_film
{
color:white;
background:#000000;
border-radius:60px;
}
#ajouter_film:hover
{
color:black;
border-radius:60px;
background:#a50000;
}
#supprimer
{
color:black;
background:#a50000;
border-radius:60px;
}
#supprimer:hover
{
color:white;
border-radius:60px;
background:#000000;
}
#pushButton_3
{
color:white;
background:#000000;
border-radius:60px;
}
#pushButton_3:hover
{
color:black;
border-radius:60px;
background:#a50000;
}
</string>
     </property>
     <property name="text">
      <string>SUPPRIMER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_chercher_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>CHERCHER</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>160</y>
       <width>351</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>AFFICHER</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>491</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>170</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TRI PAR:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_nom">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>190</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>NOM</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_duree">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>210</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>DUREE</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_language">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>230</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="imprimer">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>IMPRIMER</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>MODIFIER</string>
    </attribute>
    <widget class="QLabel" name="label_nom_film_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NOUVEAU_NOM:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_duree_film_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NOUVELLE_DUREE:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_language_film_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NOUVELLE_LANGUAGE:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_type_film_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NOUVEAU_TYPE:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_id_film_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID_FILM:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nouveau_nom">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nouveau_duree">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nouveau_language">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nouveau_type">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifier">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
font-family:century gothic;
font-size:24px;

}
QGroupBox
{
background:#333;
background-image:url(:/caca.jpg);
}
#pushButton_modifier
{
color:black;
background:#a50000;
border-radius:60px;
}
#pushButton_modifier:hover
{
color:white;
border-radius:60px;
background:#000000;
}</string>
     </property>
     <property name="text">
      <string>MODIFIER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modif">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>110</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>CHERCHER</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_5">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>160</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>CHERCHER</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>100</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
font-family:century gothic;
font-size:24px;

}
QGroupBox
{
background:#333;
background-image:url(:/caca.jpg);
}
#pushButton_3
{
color:black;
background:#a50000;
border-radius:60px;
}
#pushButton_3:hover
{
color:white;
border-radius:60px;
background:#000000;
}</string>
     </property>
     <property name="text">
      <string>chercher</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>421</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>cherrcher par ID:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
